crates/ruff_python_formatter/src/type_param/type_param_type_var_tuple.rs

```diff
@@ -8,7 +8,15 @@ pub struct FormatTypeParamTypeVarTuple;
 
 impl FormatNodeRule<TypeParamTypeVarTuple> for FormatTypeParamTypeVarTuple {
     fn fmt_fields(&self, item: &TypeParamTypeVarTuple, f: &mut PyFormatter) -> FormatResult<()> {
-        let TypeParamTypeVarTuple { range: _, name } = item;
-        write!(f, [token("*"), name.format()])
+        let TypeParamTypeVarTuple {
+            range: _,
+            name,
+            default_value,
+        } = item;
+        write!(f, [token("*"), name.format()])?;
+        if let Some(default_value) = default_value {
+            write!(f, [token("="), space(), default_value.format()])?;
+        }
```

Comment:
I think this will require a space before the `=` token as well.

```suggestion
        if let Some(default_value) = default_value {
            write!(f, [space(), token("="), space(), default_value.format()])?;
        }
```

Reply:
Agree, done